ACC2024/RobotDynamics/@RigidBody2D/test/getLagrangian.m

```diff
@@ -0,0 +1,22 @@
+function [L,T,V]= getLagrangian(bodyObjName,generalCoordinates,generalVelocities)
```

Comment:
This function is functionally correct but I would like to define it using classes. I would expect we have an "ArticulatedRobot" class. Articulated class can have multiple RigidBodys linked together and the getLagrangian is a member function of the ArticulatedRobot class. So if we have a biped robot, we can get its lagragian by bipedRobot.getLagrangian().

Reply:
Agree.